### DISCUSSION TITLE
Please allow us to contribute to the Chinese translation! 中文翻译

### DISCUSSION THREAD

**Discussion by KongNan** [netbox-community/netbox#14815] [Help Wanted!]
It's good to see that Netbox now supports multiple languages. Currently there are four languages in [transifex](https://explore.transifex.com/netbox-community/netbox/): Spanish, French, Portuguese and Russian, all translated to 100%. Please add Chinese(中文) language to transifex, [Chinese speaking users will be happy to translate](https://github.com/netbox-community/netbox/discussions/11361). Make Netbox better!

**Comment by Songxwn** [comment]
支持

**Reply by MoLin-g** [reply]
希望支持中文语言
